CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+The documentation for each repository is available in the "**docs/**" directory of that repository.
+It is written in [Markdown format] and built using [MkDocs].
+
+## Testing Your Changes Locally
+
+We **strongly** encourage you to test your changes locally before contributing them.
+To do so, you first need to do three things; these are:
+
+1. Have MkDocs installed.
+2. Have the Laminas documentation theme available to the laminas-form directory.
+3. Update `mkdocs.yml` to build the docs using the documentation theme.
+
+To complete these steps, please follow the instructions below.
+
+1. [Install MkDocs].
+2. Clone the [documentation-theme repository] locally.
+    ```
+    git clone git@github.com:laminas/documentation-theme.git documentation-theme
+    ```
+3. Copy the theme directory from the cloned documentation-theme repository to the root directory of this project.
+4. Replace lines 91 and 92 of `mkdocs.yml` with the following Yaml snippet, changing them to suit the repository that you are working with.
+    ```yaml
+    # The following example assumes that you are working with laminas-form
+    repo_url: 'https://github.com/laminas/laminas-form'
+    extra:
+        repo_name: laminas/laminas-form
+        project: Components
+        base_url: https://docs.laminas.dev/
+        project_url: https://docs.laminas.dev/components/
+    theme:
+        name: null
+        custom_dir: 'theme/'
+        static_templates:
+          - pages/404.html
+    markdown_extensions:
+        - pymdownx.superfences:
```

Comment:
I followed the instructions outlined in this document to set up a new environment and ran into the following issue.

```
INFO    -  Building documentation...
ERROR   -  Config value: 'markdown_extensions'. Error: Failed loading extension "pymdownx.superfences".

Aborted with 1 Configuration Errors!
```

After installing [PyMdown Extensions](https://facelessuser.github.io/pymdown-extensions/installation/) it worked. Is that requirement mentioned anywhere?

Reply:
@arueckauer
No, not at the moment but the guide should solve the problem.

One more extension is needed from the PyMdown Extensions: "[Tabbed](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/)"

The extension are used and needed because the underlying Markdown parser from MkDocs is limited. We use this for

* code sample in lists: https://docs.mezzio.dev/mezzio/v3/features/router/uri-generation/#naming-routes
* code sample in blockquotes: https://docs.laminas.dev/laminas-view/helpers/head-style/
* and tabs: https://docs.laminas.dev/laminas-i18n/validators/post-code/#using-locale

@settermjd
The note from @arueckauer is an important info which must be added as separate section to the guide.

The update script from the documentation build process can be used as reference:

https://github.com/laminas/documentation-theme/blob/b95cee32ad5b035a6dc7af8b3f57a387e8644900/update_mkdocs_yml.py#L19-L32